docs/using-the-compiler.rst

```diff
@@ -390,7 +390,9 @@ Input Description
         //   userdoc - User documentation (natspec)
         //   metadata - Metadata
         //   ir - Yul intermediate representation of the code before optimization
+        //   irAst - AST of Yul intermediate representation of the code before optimization in JSON format
         //   irOptimized - Intermediate representation after optimization
+        //   irOptimizedAsT - AST of intermediate representation after optimization in JSON format
```

Comment:
Well, the whole output is in JSON format :) These particular keys are just a part of it. I wouldn't mention it explicitly if it's not something unusual and unexpected, like JSON serialized into a string (which we do for metadata).

```suggestion
        //   irAst - AST of Yul intermediate representation of the code before optimization
        //   irOptimized - Intermediate representation after optimization
        //   irOptimizedAsT - AST of intermediate representation after optimization
```